flocker/node/test/test_script.py

```diff
@@ -58,13 +58,14 @@ def setup_config(test):
             u"version": 1,
         }))
     ca_set.copy_to(scratch_directory, node=True)
+    test.ca_set = ca_set
     test.non_existent_file = scratch_directory.child('missing-config.yml')
 
 deployer = object()
 
 
 def deployer_factory_stub(**kw):
-    if set(kw.keys()) != {"node_uuid", "hostname"}:
+    if set(kw.keys()) != {"node_uuid", "cluster_uuid", "hostname"}:
```

Comment:
Would be nice to formalize this somehow.  File an issue for an explicit interface for this thing and for fixing this stub to be at least minimally verified, fake-wise?